include/zephyr/dt-bindings/sensor/qdec_stm32.h

```diff
@@ -23,4 +23,8 @@
 #define FDIV32_N6	14
 #define FDIV32_N8	15
 
+#define MODE_X2_TI1  0
+#define MODE_X2_TI2  1
+#define MODE_X4_TI12 2
```

Comment:
Non blocking: adjust indentation to fit previous lines

Reply:
Not sure if I have understood; do you mean aligning the numbers?

Reply:
Oh I understood I was using tab expansion and seeing things differently.

Sorry.